Formula/dmd.rb

```diff
@@ -105,6 +105,30 @@ def install
         inreplace "posix.mak", "install: $(TOOLS) $(CURL_TOOLS)", "install: $(TOOLS) $(ROOT)/dustmite"
         system "make", "install", *make_args
       end
+    else
+      make_args.unshift "DMD_DIR=#{buildpath}", "DRUNTIME_PATH=#{buildpath}/druntime", "PHOBOS_PATH=#{buildpath}/phobos"
+      (buildpath/"druntime").install resource("druntime")
+      (buildpath/"phobos").install resource("phobos")
+      system "make", "-C", "druntime", *make_args
+      system "make", "-C", "phobos", "VERSION=#{buildpath}/VERSION", *make_args
+
+      resource("tools").stage do
+        inreplace "posix.mak", "install: $(TOOLS) $(CURL_TOOLS)", "install: $(TOOLS) $(ROOT)/dustmite"
+        system "make", "install", *make_args
+      end
+
+      (include/"dlang/dmd").install Dir["druntime/import/*"]
+      cp_r ["phobos/std", "phobos/etc"], include/"dlang/dmd"
+      lib.install Dir["druntime/lib/*", "phobos/**/libphobos2.a"]
+
+      conf = buildpath/"dmd.conf"
+      # Can't use opt_include or opt_lib here because dmd won't have been
+      # linked into opt by the time this build runs:
+      conf.write <<-EOS.undent
+        [Environment]
+        DFLAGS=-I#{include}/dlang/dmd -L-L#{lib}
```

Comment:
Is the -L-L actually intentional?

Reply:
Yes. The first `-L` means "send this argument to the linker"